dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataelement/DataElementCategoryOption.java

```diff
@@ -82,11 +87,6 @@ public DataElementCategoryOption( String name )
     // Logic
     // -------------------------------------------------------------------------
 
-    public boolean isDefault()
-    {
-        return DEFAULT_NAME.equals( name );
-    }
-

```

Comment:
I think we still need this method for other purposes ( import/export and other services ) since this default name is hardcoded.

Reply:
I decided to keep it removed. IntelliJ shows that it was not used by any other code. So I think it shouldn't be there. Can be added when later added.